perfkitbenchmarker/disk.py

```diff
@@ -18,14 +18,45 @@
 """
 
 import abc
+import logging
 
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import resource
 
+FLAGS = flags.FLAGS
+
+
+# These are the (deprecated) old disk type names
 STANDARD = 'standard'
 REMOTE_SSD = 'remote_ssd'
 PIOPS = 'piops'  # Provisioned IOPS (SSD) in AWS
+
+# 'local' refers to disks that come attached to VMs. It is the only
+# "universal" disk type that is not associated with a provider. It
+# exists because we can provision a local disk without creating a disk
```

Comment:
would be nice to enforce the requirement that a disk spec is needed to provision any disk! then we can get rid off this special "local" thing. 

Not in this CL, but would be good to file a issue to track this.